modules/core/src/lib/tooltip.js

```diff
@@ -0,0 +1,53 @@
+/* global document */
+
+// const TOOLTIP_EMPTY_STYLE = {
+//   display: 'none'
+// };
+// const TOOLTIP_STYLE = {
+//   'font-family': 'Helvetica, Arial, sans-serif',
+//   position: 'absolute',
+//   padding: '4px',
+//   margin: '8px',
+//   background: 'rgba(0, 0, 0, 0.8)',
+//   color: '#fff',
+//   'max-width': '300px',
+//   'font-size': '10px',
+//   'z-index': 9,
+//   'pointer-events': 'none'
+// };
+// TODO where should the CSS be added?
+
+const buildTable = (jsonObject, shouldIndent) => {
```

Comment:
Nit: For big functions like this I find the traditional `function` keyword syntax easier to read. IMHO Arrow functions are best for inline functions, one line functions etc.